src/actions/actionTypes.ts

```diff
@@ -15,7 +15,17 @@ export const GENERATE_LZ_STRING = 'GENERATE_LZ_STRING'
 export const HANDLE_CONSOLE_LOG = 'HANDLE_CONSOLE_LOG'
 export const EVAL_INTERPRETER_SUCCESS = 'EVAL_INTERPRETER_SUCCESS'
 export const EVAL_INTERPRETER_ERROR = 'EVAL_INTERPRETER_ERROR'
-export const INTERRUPT_EXECUTION = 'INTERRUPT_EXECUTION'
+/**
+ * Called to signal the beginning of an interruption,
+ * i.e the trigger for the interpreter to interrupt execution.
+ */
+export const BEGIN_INTERRUPT_EXECUTION = 'BEGIN_INTERRUPT_EXECUTION'
+/**
+ * Called to signal the end of an interruption,
+ * i.e called after the interpreter is told to stop interruption,
+ * to cause UI changes.
+ */
```

Comment:
Can we move these comments to the reducers? I think there are at least three places we can comment on the actions:

- actionTypes
- actionCreators
- reducers & saga

And I think we should standardise them being in the reducers and saga because

1. Most of them are already there, and
2. That's where the logic is